applicationinsights.ts

```diff
@@ -205,12 +206,14 @@ export class Configuration {
     /**
      * Sets the state of automatic dependency correlation (enabled by default)
      * @param value if true dependencies will be correlated with requests
+     * @param useAsyncHooks if true, forces use of experimental async_hooks module to provide correlation. If false, instead uses async-listener. If left blank, the best option is chosen for you based on your version on NodeJS.
```

Comment:
Nit: "on your version on NodeJS" => "on your version **of** Node.js"
Also consider "instead uses async-listener" => "instead uses only patching-based techniques" or something along those lines in case we'd like to replace `async-listener` in particular.

Reply:
Updated